diff --git a/CMakeLists.txt b/CMakeLists.txt
index 657a1f4..f2778e1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -70,7 +70,7 @@
 ## - GFLAGS_INSTALL_SHARED_LIBS
 ## - GFLAGS_INSTALL_STATIC_LIBS
 
-cmake_minimum_required (VERSION 3.0.2 FATAL_ERROR)
+cmake_minimum_required (VERSION 3.26.3 FATAL_ERROR)
 
 if (POLICY CMP0042)
   cmake_policy (SET CMP0042 NEW)
diff --git a/test/config/CMakeLists.txt b/test/config/CMakeLists.txt
index 6190b25..82fac14 100644
--- a/test/config/CMakeLists.txt
+++ b/test/config/CMakeLists.txt
@@ -1,6 +1,6 @@
 ## gflags package configuration tests
 
-cmake_minimum_required (VERSION 2.8.12 FATAL_ERROR)
+cmake_minimum_required (VERSION 3.26.3 FATAL_ERROR)
 
 project (gflags_${TEST_NAME})
 
diff --git a/test/nc/CMakeLists.txt b/test/nc/CMakeLists.txt
index d00b07d..ea91ee4 100644
--- a/test/nc/CMakeLists.txt
+++ b/test/nc/CMakeLists.txt
@@ -1,6 +1,6 @@
 ## gflags negative compilation tests
 
-cmake_minimum_required (VERSION 2.8.12 FATAL_ERROR)
+cmake_minimum_required (VERSION 3.26.3 FATAL_ERROR)
 
 if (NOT TEST_NAME)
   message (FATAL_ERROR "Missing TEST_NAME CMake flag")
