diff -Naur old/ipbus-software-2.6.4/controlhub/Makefile new/ipbus-software-2.6.4/controlhub/Makefile
--- old/ipbus-software-2.6.4/controlhub/Makefile	2019-07-29 17:38:26.000000000 +0200
+++ new/ipbus-software-2.6.4/controlhub/Makefile	2019-09-19 04:40:21.228790668 +0200
@@ -147,24 +147,24 @@
 install:
 	cd scripts; find . -name "controlhub_*" -exec install -D -m 755 {} $(bindir)/{} \;
 	mkdir -p $(libdir) && cp -r rel/controlhub $(libdir)/
-	install -D -m 644 pkg/rsyslog.d.conf /etc/rsyslog.d/controlhub.conf
-	install -D -m 644 pkg/logrotate.d.conf /etc/logrotate.d/controlhub.conf
+	#install -D -m 644 pkg/rsyslog.d.conf /etc/rsyslog.d/controlhub.conf
+	#install -D -m 644 pkg/logrotate.d.conf /etc/logrotate.d/controlhub.conf
 
-	mkdir -p /etc/controlhub/default
-	mv $(libdir)/controlhub/sys.config /etc/controlhub/default/sys.config
-	sed -i "s|\".*core.config\"|\"$(libdir)/controlhub/core.config\"|" /etc/controlhub/default/sys.config
-	sed -i "s|CONTROLHUB_CONFIG_FILE_DEFAULT=.*|CONTROLHUB_CONFIG_FILE_DEFAULT=/etc/controlhub/default/sys.config|" $(libdir)/controlhub/bin/controlhub
-	sed -i "s|CONTROLHUB_CONFIG_FILE_OPTIONAL=.*|CONTROLHUB_CONFIG_FILE_OPTIONAL=/etc/controlhub/sys.config|" $(libdir)/controlhub/bin/controlhub
+	#mkdir -p /etc/controlhub/default
+	#mv $(libdir)/controlhub/sys.config /etc/controlhub/default/sys.config
+	#sed -i "s|\".*core.config\"|\"$(libdir)/controlhub/core.config\"|" /etc/controlhub/default/sys.config
+	#sed -i "s|CONTROLHUB_CONFIG_FILE_DEFAULT=.*|CONTROLHUB_CONFIG_FILE_DEFAULT=/etc/controlhub/default/sys.config|" $(libdir)/controlhub/bin/controlhub
+	#sed -i "s|CONTROLHUB_CONFIG_FILE_OPTIONAL=.*|CONTROLHUB_CONFIG_FILE_OPTIONAL=/etc/controlhub/sys.config|" $(libdir)/controlhub/bin/controlhub
 
-	sed -i "s|CONTROLHUB_BIN_DIR=.*|CONTROLHUB_BIN_DIR=$(libdir)/controlhub/bin|" $(bindir)/controlhub_*
+	#sed -i "s|CONTROLHUB_BIN_DIR=.*|CONTROLHUB_BIN_DIR=$(libdir)/controlhub/bin|" $(bindir)/controlhub_*
 
-	mkdir -p /var/log/controlhub
-	touch /var/log/controlhub/controlhub.log
-	chmod 644 /var/log/controlhub/controlhub.log
+	#mkdir -p /var/log/controlhub
+	#touch /var/log/controlhub/controlhub.log
+	#chmod 644 /var/log/controlhub/controlhub.log
 ifeq (,$(shell /bin/bash -c "command -v service"))
-	@echo " WARNING : 'service' command not detected! After the installation has finished, you should restart rsyslog to ensure that the ControlHub syslog config is loaded"
+	#@echo " WARNING : 'service' command not detected! After the installation has finished, you should restart rsyslog to ensure that the ControlHub syslog config is loaded"
 else
-	service rsyslog restart
+	#service rsyslog restart
 endif
 
 uninstall:
diff -Naur old/ipbus-software-2.6.4/uhal/pycohal/Makefile new/ipbus-software-2.6.4/uhal/pycohal/Makefile
--- old/ipbus-software-2.6.4/uhal/pycohal/Makefile	2019-07-29 17:38:26.000000000 +0200
+++ new/ipbus-software-2.6.4/uhal/pycohal/Makefile	2019-09-19 04:39:11.169248745 +0200
@@ -54,7 +54,6 @@
 		pugixml \
 		\
 		boost_filesystem \
-		${EXTERN_BOOST_PYTHON_LIB} \
 		boost_regex \
 		boost_system \
 		boost_thread \
diff -Naur old/ipbus-software-2.6.4/uhal/uhal/include/uhal/ClientFactory.hpp new/ipbus-software-2.6.4/uhal/uhal/include/uhal/ClientFactory.hpp
--- old/ipbus-software-2.6.4/uhal/uhal/include/uhal/ClientFactory.hpp	2019-07-29 17:38:26.000000000 +0200
+++ new/ipbus-software-2.6.4/uhal/uhal/include/uhal/ClientFactory.hpp	2019-09-19 04:38:26.690904695 +0200
@@ -46,6 +46,7 @@
 
 #include <boost/shared_ptr.hpp>
 #include <boost/unordered_map.hpp>
+#include <boost/core/noncopyable.hpp>
 
 #include <map>
 
diff -Naur old/ipbus-software-2.6.4/uhal/uhal/include/uhal/ConnectionManager.hpp new/ipbus-software-2.6.4/uhal/uhal/include/uhal/ConnectionManager.hpp
--- old/ipbus-software-2.6.4/uhal/uhal/include/uhal/ConnectionManager.hpp	2019-07-29 17:38:26.000000000 +0200
+++ new/ipbus-software-2.6.4/uhal/uhal/include/uhal/ConnectionManager.hpp	2019-09-19 04:38:46.100054831 +0200
@@ -45,6 +45,7 @@
 #include "uhal/HwInterface.hpp"
 
 #include <boost/filesystem/path.hpp>
+#include <boost/core/noncopyable.hpp>
 
 #include <vector>
 #include <set>
