diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8d440f3..264ee51 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -24,10 +24,10 @@
 # LIABILITY, ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE
 # OF THIS SOFTWARE.
 
-cmake_minimum_required(VERSION 3.9.0)
+cmake_minimum_required(VERSION 3.26.3)
 
 # Default policy is from 3.9.0
-cmake_policy(VERSION 3.9.0)
+cmake_policy(VERSION 3.26.3)
 
 # Find CMake modules in cmake/
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
diff --git a/doc/CMakeLists.txt b/doc/CMakeLists.txt
index 6cc3d2e..c981ddf 100644
--- a/doc/CMakeLists.txt
+++ b/doc/CMakeLists.txt
@@ -24,7 +24,7 @@
 # LIABILITY, ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE
 # OF THIS SOFTWARE.
 
-cmake_minimum_required(VERSION 3.2.0)
+cmake_minimum_required(VERSION 3.26.3)
 
 include(Sphinx)
 
