diff -Naur old/ipbus-software-2.8.1/controlhub/Makefile new/ipbus-software-2.8.1/controlhub/Makefile
--- old/ipbus-software-2.8.1/controlhub/Makefile	2021-01-29 01:18:00.000000000 +0000
+++ new/ipbus-software-2.8.1/controlhub/Makefile	2021-01-31 05:54:26.109817120 +0000
@@ -159,27 +159,27 @@
 install:
 	cd scripts; find . -name "controlhub_*" -exec install -D -m 755 {} $(bindir)/{} \;
 	mkdir -p $(libdir) && cp -r rel/controlhub $(libdir)/
-	install -D -m 644 pkg/rsyslog.d.conf /etc/rsyslog.d/controlhub.conf
-	install -D -m 644 pkg/logrotate.d.conf /etc/logrotate.d/controlhub.conf
+	#install -D -m 644 pkg/rsyslog.d.conf /etc/rsyslog.d/controlhub.conf
+	#install -D -m 644 pkg/logrotate.d.conf /etc/logrotate.d/controlhub.conf
 
-	mkdir -p /etc/controlhub/default
-	mv $(libdir)/controlhub/sys.config /etc/controlhub/default/sys.config
-	sed -i "s|\".*core.config\"|\"$(libdir)/controlhub/core.config\"|" /etc/controlhub/default/sys.config
-	sed -i "s|CONTROLHUB_CONFIG_FILE_DEFAULT=.*|CONTROLHUB_CONFIG_FILE_DEFAULT=/etc/controlhub/default/sys.config|" $(libdir)/controlhub/bin/controlhub
-	sed -i "s|CONTROLHUB_CONFIG_FILE_OPTIONAL=.*|CONTROLHUB_CONFIG_FILE_OPTIONAL=/etc/controlhub/sys.config|" $(libdir)/controlhub/bin/controlhub
+	#mkdir -p /etc/controlhub/default
+	#mv $(libdir)/controlhub/sys.config /etc/controlhub/default/sys.config
+	#sed -i "s|\".*core.config\"|\"$(libdir)/controlhub/core.config\"|" /etc/controlhub/default/sys.config
+	#sed -i "s|CONTROLHUB_CONFIG_FILE_DEFAULT=.*|CONTROLHUB_CONFIG_FILE_DEFAULT=/etc/controlhub/default/sys.config|" $(libdir)/controlhub/bin/controlhub
+	#sed -i "s|CONTROLHUB_CONFIG_FILE_OPTIONAL=.*|CONTROLHUB_CONFIG_FILE_OPTIONAL=/etc/controlhub/sys.config|" $(libdir)/controlhub/bin/controlhub
 
 	sed -i "s|CONTROLHUB_BIN_DIR=.*|CONTROLHUB_BIN_DIR=$(libdir)/controlhub/bin|" $(bindir)/controlhub_*
 
-	mkdir -p /var/log/controlhub
-	touch /var/log/controlhub/controlhub.log
-	chmod 644 /var/log/controlhub/controlhub.log
-ifeq (,$(shell /bin/bash -c "command -v service"))
-	@echo " WARNING : 'service' command not detected! After the installation has finished, you should restart rsyslog to ensure that the ControlHub syslog config is loaded"
-else
-	-service rsyslog restart
-endif
+	#mkdir -p /var/log/controlhub
+	#touch /var/log/controlhub/controlhub.log
+	#chmod 644 /var/log/controlhub/controlhub.log
+#ifeq (,$(shell /bin/bash -c "command -v service"))
+	#@echo " WARNING : 'service' command not detected! After the installation has finished, you should restart rsyslog to ensure that the ControlHub syslog config is loaded"
+#else
+#	-service rsyslog restart
+#endif
 
 uninstall:
 	rm -rf $(bindir)/controlhub*
 	rm -rf $(libdir)/controlhub
-	rm -rf /etc/rsyslog.d/controlhub.conf /etc/logrotate.d/controlhub.conf /etc/controlhub/default
+	#rm -rf /etc/rsyslog.d/controlhub.conf /etc/logrotate.d/controlhub.conf /etc/controlhub/default
diff -Naur old/ipbus-software-2.8.1/uhal/python/Makefile new/ipbus-software-2.8.1/uhal/python/Makefile
--- old/ipbus-software-2.8.1/uhal/python/Makefile	2021-01-29 01:18:00.000000000 +0000
+++ new/ipbus-software-2.8.1/uhal/python/Makefile	2021-01-31 05:55:33.256950919 +0000
@@ -27,7 +27,7 @@
 
 
 IncludePaths = include  \
-		pybind11/include \
+		${EXTERN_PYBIND11_INCLUDE_PREFIX} \
 		${EXTERN_BOOST_INCLUDE_PREFIX} \
 		${EXTERN_PUGIXML_INCLUDE_PREFIX} \
 		${PYTHON_INCLUDE_PREFIX} \
