diff --git a/pyproject.toml b/pyproject.toml
index 26d299a..e1d7810 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -28,8 +28,6 @@ classifiers = [
   "Programming Language :: Python :: 3.12",
   "Programming Language :: Python :: Implementation :: CPython",
   "Programming Language :: Python :: Implementation :: PyPy",
-  "Topic :: File Formats :: JSON",
-  "Topic :: File Formats :: JSON :: JSON Schema",
 ]
 dynamic = ["version", "readme"]
 
