diff -Naur old/cpr-1.5.2/cpr/session.cpp new/cpr-1.5.2/cpr/session.cpp
--- old/cpr-1.5.2/cpr/session.cpp	2020-10-20 13:24:44.000000000 +0000
+++ new/cpr-1.5.2/cpr/session.cpp	2020-12-07 17:31:47.046353457 +0000
@@ -373,7 +373,7 @@
 #endif
         curl_easy_setopt(curl, CURLOPT_SSL_VERIFYPEER, opts.verify_peer ? ON : OFF);
         curl_easy_setopt(curl, CURLOPT_SSL_VERIFYHOST, opts.verify_host ? 2L : 0L);
-        curl_easy_setopt(curl, CURLOPT_SSL_VERIFYSTATUS, opts.verify_status ? ON : OFF);
+        //curl_easy_setopt(curl, CURLOPT_SSL_VERIFYSTATUS, opts.verify_status ? ON : OFF);
         curl_easy_setopt(curl, CURLOPT_SSLVERSION,
                          opts.ssl_version
 #if SUPPORT_MAX_TLS_VERSION
diff -Naur old/cpr-1.5.2/cpr-config.cmake new/cpr-1.5.2/cpr-config.cmake
--- old/cpr-1.5.2/cpr-config.cmake	2020-10-20 13:24:44.000000000 +0000
+++ new/cpr-1.5.2/cpr-config.cmake	2020-12-07 17:32:30.385786058 +0000
@@ -11,10 +11,12 @@
 #                     file that uses this interface
 
 find_path(CPR_INCLUDE_DIR
-          NAMES cpr.h)
+          NAMES cpr/cpr.h
+          PATHS ENV CPR_FQ_DIR)
 
 find_library(CPR_LIBRARY
              NAMES cpr
+             PATHS ENV CPR_FQ_DIR
              HINTS ${CPR_LIBRARY_ROOT})
 
 include(FindPackageHandleStandardArgs)
