diff --git a/CMakeLists.txt b/CMakeLists.txt
index 70acb69..726bb3e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1468,7 +1468,7 @@ install(FILES ${HDR_COMPAT}
 install(FILES
         ${PROJECT_BINARY_DIR}/${CMAKE_FILES_DIRECTORY}/LibeventConfig.cmake
         ${PROJECT_BINARY_DIR}/LibeventConfigVersion.cmake
-        DESTINATION "${EVENT_INSTALL_CMAKE_DIR}"
+        DESTINATION "${DEF_INSTALL_CMAKE_DIR}"
         COMPONENT dev)
 
 # Install exports for the install-tree.
