diff --git a/CMake/folly-config.cmake.in b/CMake/folly-config.cmake.in
index 1689f9a..3682f3b 100644
--- a/CMake/folly-config.cmake.in
+++ b/CMake/folly-config.cmake.in
@@ -29,6 +29,13 @@ set(FOLLY_LIBRARIES Folly::folly)
 
 # Find folly's dependencies
 find_dependency(fmt)
+find_dependency(glog)
+find_dependency(double-conversion)
+find_dependency(OpenSSL)
+find_dependency(LibLZMA)
+find_dependency(BZip2)
+find_dependency(Libevent)
+find_dependency(zstd COMPONENTS libzstd_shared REQUIRED)
 
 set(Boost_USE_STATIC_LIBS "@FOLLY_BOOST_LINK_STATIC@")
 find_dependency(Boost 1.51.0 MODULE
diff --git a/CMake/folly-deps.cmake b/CMake/folly-deps.cmake
index e0e02c0..930d331 100644
--- a/CMake/folly-deps.cmake
+++ b/CMake/folly-deps.cmake
@@ -48,7 +48,7 @@ find_package(Boost 1.51.0 MODULE
 list(APPEND FOLLY_LINK_LIBRARIES ${Boost_LIBRARIES})
 list(APPEND FOLLY_INCLUDE_DIRECTORIES ${Boost_INCLUDE_DIRS})
 
-find_package(DoubleConversion MODULE REQUIRED)
+find_package(double-conversion CONFIG REQUIRED)
 list(APPEND FOLLY_LINK_LIBRARIES ${DOUBLE_CONVERSION_LIBRARY})
 list(APPEND FOLLY_INCLUDE_DIRECTORIES ${DOUBLE_CONVERSION_INCLUDE_DIR})
 
@@ -111,7 +111,7 @@ if (LZ4_FOUND)
   list(APPEND FOLLY_LINK_LIBRARIES ${LZ4_LIBRARY})
 endif()
 
-find_package(Zstd MODULE)
+find_package(zstd CONFIG)
 set(FOLLY_HAVE_LIBZSTD ${ZSTD_FOUND})
 if(ZSTD_FOUND)
   list(APPEND FOLLY_INCLUDE_DIRECTORIES ${ZSTD_INCLUDE_DIR})
@@ -133,9 +133,10 @@ find_package(Libiberty)
 list(APPEND FOLLY_LINK_LIBRARIES ${LIBIBERTY_LIBRARIES})
 list(APPEND FOLLY_INCLUDE_DIRECTORIES ${LIBIBERTY_INCLUDE_DIRS})
 
-find_package(LibAIO)
-list(APPEND FOLLY_LINK_LIBRARIES ${LIBAIO_LIBRARIES})
-list(APPEND FOLLY_INCLUDE_DIRECTORIES ${LIBAIO_INCLUDE_DIRS})
+#find_package(LibAIO)
+#list(APPEND FOLLY_LINK_LIBRARIES ${LIBAIO_LIBRARIES})
+#list(APPEND FOLLY_INCLUDE_DIRECTORIES ${LIBAIO_INCLUDE_DIRS})
+set(LIBAIO_FOUND FALSE)
 
 find_package(LibUring)
 list(APPEND FOLLY_LINK_LIBRARIES ${LIBURING_LIBRARIES})
@@ -304,12 +305,34 @@ if (NOT DEFINED fmt_CONFIG)
     # Fallback on a normal search on the current system
     find_package(Fmt MODULE REQUIRED)
 endif()
-target_link_libraries(folly_deps INTERFACE fmt::fmt)
-
-list(REMOVE_DUPLICATES FOLLY_INCLUDE_DIRECTORIES)
-target_include_directories(folly_deps INTERFACE ${FOLLY_INCLUDE_DIRECTORIES})
-target_link_libraries(folly_deps INTERFACE
-  ${FOLLY_LINK_LIBRARIES}
-  ${FOLLY_SHINY_DEPENDENCIES}
-  ${FOLLY_ASAN_FLAGS}
+#target_link_libraries(folly_deps INTERFACE fmt::fmt)
+
+#list(REMOVE_DUPLICATES FOLLY_INCLUDE_DIRECTORIES)
+#target_include_directories(folly_deps INTERFACE ${FOLLY_INCLUDE_DIRECTORIES})
+#target_link_libraries(folly_deps INTERFACE
+#  ${FOLLY_LINK_LIBRARIES}
+#  ${FOLLY_SHINY_DEPENDENCIES}
+#  ${
+#)
+
+# libevent doesn't provide a target with an includes interface, so we do this manually
+#target_include_directories(folly_deps PRIVATE ${LIBEVENT_INCLUDE_DIR})
+target_include_directories(folly_deps
+  INTERFACE
+    $<BUILD_INTERFACE:${LIBEVENT_INCLUDE_DIR}>
 )
+target_link_libraries(folly_deps INTERFACE Boost::context
+                                           Boost::filesystem
+                                           Boost::program_options
+                                           Boost::regex
+                                           Boost::system
+                                           Boost::thread
+                                           fmt::fmt
+                                           double-conversion::double-conversion
+                                           glog::glog
+                                           event_shared
+                                           OpenSSL::SSL
+                                           BZip2::BZip2
+                                           LibLZMA::LibLZMA
+                                           zstd::libzstd_shared
+                                           )
diff --git a/folly/CMakeLists.txt b/folly/CMakeLists.txt
index 883f27c..6e27b6a 100644
--- a/folly/CMakeLists.txt
+++ b/folly/CMakeLists.txt
@@ -28,7 +28,7 @@ install(
 )
 
 add_subdirectory(experimental/exception_tracer)
-add_subdirectory(logging/example)
+#add_subdirectory(logging/example)
 
 if (PYTHON_EXTENSIONS)
   # Create tree of symbolic links in structure required for successful
